### ISSUE TITLE
Cannot convert to integer 

### ISSUE THREAD

**Issue by carlokok** [commercetools/commercetools-dotnet-core-sdk#155]
When I do a query that requests the aggregate for variants.price.centAmount, I get back:
```
"variants.price.centAmount":{"type":"range","dataType":"number","ranges":[{"type":"double","from":0.0,"fromStr":"0.0","to":0.0,"toStr":"","count":119104,"totalCount":119104,"total":3.715960759E9,"min":0.0,"max":2301000.0,"mean":31199.294389777002}]}
```

Which results in:
Newtonsoft.Json.JsonReaderException: 'Could not convert to integer: 3715960759. Path 'ranges[0].total', line 1, position 28123.'

**Comment by jenschude** [comment]
Would it be possible to get the original request and response? Never seen a number this high actually. And I'm always thought exponential presentation shouldn't happen on API level.

**Comment by carlokok** [comment]
@jenschude Is there a way to provide this privately?

**Comment by jenschude** [comment]
Sure. Company mail can be seen at my profile page ;)

**Comment by jenschude** [comment]
Or sent in a mail to support@commercetools.com

**Comment by jenschude** [comment]
Found an issue in the SDK itself and fixed it with e47a4ff

Will cut a release today.

Thanks for the report
